app/models/saved_yardsale.rb

```diff
@@ -0,0 +1,3 @@
+class SavedYardsale < ApplicationRecord
+  belongs_to :user
```

Comment:
I think this also belong to yard_sale.
You also need to specify the name of the table that was created with the migration. Was it user_yardsales? I can't remember

Reply:
I think we still need to specify the name of the table manually for this model. https://stackoverflow.com/questions/4613574/how-do-i-explicitly-specify-a-models-table-name-mapping-in-rails